### ISSUE TITLE
Error with abaco list

### ISSUE THREAD

**Issue by mwvaughn** [johnfonner/abaco-cli#2]
I get the following on Mac OS X 10.12.6 (16G29) / jq-1.4 when I do an `abaco list`

```
abaco list
jq: error: tsv is not a valid format
jq: error: tsv is not a valid format
```

**Comment by jturcino** [comment]
The `@tsv` functionality was first included with [jq-1.5](https://stedolan.github.io/jq/manual/v1.5/). I have changed all `@tsv` references in the CLI to be compatible with [jq-1.4](https://stedolan.github.io/jq/manual/v1.4/). The commands changed were `abaco-create`, `abaco-list`, `abaco-executions`, and `abaco-workers`. The change was the following:

**Before:**
```
echo $json | jq -r '.result | [.name, .id] | @tsv' | column -t
```

**After:**
```
echo $json | | jq -r '.result | [.name, .id] |  "\(.[0]) \(.[1])"' | column -t
```
There should be no difference in the output of these commands.

**Comment by mwvaughn** [comment]
That fixed it!
